src/stories/alert.stories.js

```diff
@@ -1,13 +1,45 @@
 import '../components/atoms/Alert/cod-alert';
-import { COMMON_STORY_ARGS } from '../shared/js/storybook/args-utils';
 
 export default {
   title: 'Components/Atoms/Alert',
   argTypes: {
-    icon: COMMON_STORY_ARGS.icon,
-    iconOrder: COMMON_STORY_ARGS.order,
-    iconSize: COMMON_STORY_ARGS.longSize,
-    backgroundColor: COMMON_STORY_ARGS.bootstrapColor,
+    icon: {
+      control: { type: 'select' },
+      options: [
+        'house',
+        'house-fill',
+        'exclamation-circle',
+        'exclamation-circle-fill',
+        'exclamation-triangle',
+        'check-circle',
+        'check-circle-fill',
+      ],
+    },
+    iconOrder: {
+      control: { type: 'select' },
+      options: ['left', 'right'],
+    },
+    iconSize: {
+      control: { type: 'select' },
+      options: ['small', 'medium', 'large', 'x-large'],
+    },
+    backgroundColor: {
+      control: { type: 'select' },
+      options: [
+        'primary',
+        'secondary',
+        'success',
+        'info',
+        'warning',
+        'danger',
+        'light',
+        'dark',
+      ],
+    },
```

Comment:
This is also from an incorrect conflict resolution from dev merge. Please revert these changes.

Reply:
I was able to revert Alert.js but alert.stories.js is not letting me do anything @maxatdetroit

Reply:
This is the code that was already in this file @maxatdetroit

Reply:
<img width="340" alt="Screenshot 2024-05-06 at 2 03 07 PM" src="https://github.com/CityOfDetroit/COD-Design-System/assets/141044304/847794d4-e3b7-4688-b1a8-068638ed38fd">

Reply:
I copied the code when I was re-building the sandbox trying to understand. This is what was already there before I tried to fix the merge conflict

Reply:
> This is what was already there before I tried to fix the merge conflict

That's right, because the branch had old code that wasn't up-to-date with `dev`. Then, when we merged in `dev`, it tried to update this code to what is on dev (i.e. the newer code). We should have taken that newer code instead of keeping the old code.

Here's the new code from dev: https://is.gd/rXc37Q

That's what this section should look like (with the addition of your new storybook arg you added in this PR).